src/bundle/Resources/public/js/scripts/sidebar/extra.actions.js

```diff
@@ -56,15 +56,26 @@
             focusElement.focus();
         }
     };
+    const init = (dataset) => {
```

Comment:
I would call this function more self-explanatory name, but truth be told, I have no idea how, `autoOpenSidebar`? not exactly there, as it's conditional auto-open :thinking: